bin/oneoff/aichat_logs_to_csv

```diff
@@ -76,7 +76,7 @@ def main
           *base_row,
           message["status"],
           message["role"],
-          message["content"]
+          message["content"] || message["chatMessageText"]
```

Comment:
This is fine, but just confirming - if the `message["content"]` is falsy, this'll still fall through to `message["chatMessageText"]`.

So we can't have `message["content"]` that is things like `0` or `""` or `"0"` (though, amusingly, we still _would_ allow `" "`).

Anyway, so long as that's the case, then it's good. Otherwise if we'd want to display it, then it should hop to a `??`.

(And if we want to prevent the blanking string cases, then maybe a regex like `/\S/` in a ternary)

Reply:
Thanks for raising this, I hadn't really thought it through (sort of a last minute Friday afternoon change here). Did a little Ruby digging, and I think we're actually ok here? Does this look right to you?

<img width="277" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/25372625/7f911432-5322-4539-a50e-83d04dac1e6c">

Reply:
Also - I don't think `??` operator is available in Ruby.